pinot-hadoop/src/main/java/org/apache/pinot/hadoop/job/JobConfigConstants.java

```diff
@@ -57,4 +57,6 @@
   public static final String ENABLE_PARTITIONING = "enable.partitioning";
   public static final String ENABLE_SORTING = "enable.sorting";
   public static final String ENABLE_RESIZING = "enable.resizing";
+
+  public static final String IS_DELETE_EXTRA_SEGMENTS = "is.delete.extra.segments";
```

Comment:
It'd be good to add some descriptions to this setting on how/when to use it.

Reply:
`delete.extra.segments`